src/app/zap-templates/zcl/data-model/chip/concentration-measurement-cluster.xml

```diff
@@ -57,56 +59,63 @@ limitations under the License.
         </optionalConform>
       </feature>
     </features>
-
     <attribute side="server" code="0x0000" name="MeasuredValue" define="MEASURED_VALUE" type="single" isNullable="true" optional="true">
+      <description>MeasuredValue</description>
```

Comment:
This seems like an alchemy issue, but why do we have both "name" and "description" bits here?

Reply:
Yes, this is something that was fixed with the latest Alchemy version, will regenerate this after solving the issue with the CI that is failing at the moment.